From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: name the unused RK808 outputs

LDO_REG1/2/5/7 and SWITCH_REG2 get a name and no other constraint, so
the regulator core may switch them off as unused.

Upstream: none.

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 .../boot/dts/rockchip/rk3399-kobol-helios64.dts    | 25 ++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -511,6 +511,16 @@ &i2c0 {
 				};
 			};
 
+			/* not used */
+			vcc1v8_dvp: LDO_REG1 {
+				regulator-name = "vcc1v8_dvp";
+			};
+
+			/* not used */
+			vcc3v0_touch: LDO_REG2 {
+				regulator-name = "vcc3v0_touch";
+			};
+
 			vcc1v8_s3: LDO_REG3 {
 				regulator-name = "vcc1v8_s3";
 				regulator-always-on;
@@ -536,6 +546,11 @@ &i2c0 {
 				};
 			};
 
+			/* not used */
+			vcca3v0_codec: LDO_REG5 {
+				regulator-name = "vcca3v0_codec";
+			};
+
 			vcc1v5_s3: LDO_REG6 {
 				regulator-name = "vcc1v5_s3";
 				regulator-always-on;
@@ -548,6 +563,11 @@ &i2c0 {
 				};
 			};
 
+			/* not used */
+			vcca1v8_codec: LDO_REG7 {
+				regulator-name = "vcca1v8_codec";
+			};
+
 			vcc3v0_s3: LDO_REG8 {
 				regulator-name = "vcc3v0_s3";
 				regulator-always-on;
@@ -571,6 +591,11 @@ &i2c0 {
 					regulator-off-in-suspend;
 				};
 			};
+
+			/* not used */
+			vcc3v3_s0: SWITCH_REG2 {
+				regulator-name = "vcc3v3_s0";
+			};
 		};
 	};
 
-- 
Armbian

